applications/adv_networking_bench/README.md

```diff
@@ -1,48 +1,54 @@
+
+This is a sample application to measure a lower bound on performance for the [Advanced Network
+library](/operators/advanced_network/README.md) by receiving packets, optionally doing work on them, and freeing
+the buffers. While only freeing the packets is an unrealistic workload, it's useful to see at a high level whether
+the application is able to keep up with the bare minimum amount of work to do. The application contains both a
+transmitter and receiver that are designed to run on different systems, and may be configured independently.
 
 The performance of this application depends heavily on a properly-configured system and choosing the best
 tuning parameters that are acceptable for the workload. To configure the system please see the documentation
-for the advanced network operator. With the system tuned, the application performance will be dictated
-by batching size and whether GPUDirect is enabled.
+[here](/tutorials/high_performance_networking/README.md). With the system tuned, the application performance
+will be dictated by batching size and whether GPUDirect is enabled.
 
 At this time both the transmitter and receiver are written to handle an Ethernet+IP+UDP packet with a
 configurable payload. Other modes may be added in the future. Also, for simplicity, the transmitter and
-receiver must be configured to a single packet size.
+receiver are configured to a single packet size.
 
 ## Transmit
 
 The transmitter sends a UDP packet with an incrementing sequence of bytes after the UDP header. The batch
-size configured dictates how many packets the benchmark operator sends to the advanced network operator
-in each tick. Typically with the same number of CPU cores the transmitter will run faster than the receiver,
-so this parameter may be used to throttle the sender somewhat by making the batches very small.
+size configured dictates how many packets the benchmark operator sends to the NIC in each tick. Typically with
+the same number of CPU cores the transmitter will run faster than the receiver, so this parameter may be used
+to throttle the sender somewhat by making the batches very small.
 
 ## Receiver
 
-The receiver receives the UDP packets in either CPU-only mode or header-data split mode. CPU-only mode
-will receive the packets in CPU memory, copy the payload contents to a host-pinned staging buffer, and
-freed. In header-data split mode the user may configure a split point where the bytes before that point
-are sent to the CPU, and all bytes afterwards are sent to the GPU. Header-data split should achieve higher
-rates than CPU mode since the amount of data to the CPU can be orders of magnitude lower compared to running
-in CPU-only mode.
+The receiver receives the UDP packets in either CPU-only mode, header-data split mode, or GPU-only mode.
+- CPU-only mode will receive the packets in CPU memory, copy the payload contents to a host-pinned staging buffer, and
+  freed.
```

Comment:
"freed" sounds odd here. Maybe "free the buffers"?

Reply:
100%, though that was written before. Not sure why I changed this section.